dashboard/test/ui/features/projects.feature

```diff
@@ -44,9 +44,8 @@ Scenario: Applab Flow
   And I wait to see "#footerDiv"
   Then I should see title "Code Ninja - App Lab"
   And element "#codeWorkspace" is hidden
-  And I press the first "#footerDiv .more-link" element
   And I make all links open in the current tab
-  And I press a button with xpath "//div[@id = 'footerDiv']//a[text() = 'How It Works']"
```

Comment:
It is, but this scenario specifically checks an App Lab share.  How about I add a Game Lab scenario here that tries the "How It Works" button?